test/unit/stream_alert_rule_processor/test_config.py

```diff
@@ -18,15 +18,29 @@
 # specific test: nosetests -v -s test/unit/file.py:TestStreamPayload.test_name
 
 from collections import namedtuple
+from mock import mock_open, patch
 
 from nose.tools import assert_equal, raises
 
 from stream_alert.rule_processor.config import (
     ConfigError,
+    load_config,
+    load_env,
     validate_config,
-    load_env
 )
 
+@raises(ConfigError)
+def test_load_config_invalid():
+    """Config Validator - Load Config - Invalid"""
+    m = mock_open()
+    with patch('__builtin__.open', m, create=True):
```

Comment:
we should remove the argument to `load_config` (it was originally added for tests)